gen/calc.py

```diff
@@ -163,6 +163,11 @@ def validate_mesos_log_retention_mb(mesos_log_retention_mb):
     assert int(mesos_log_retention_mb) >= 1024, "Must retain at least 1024 MB of logs"
 
 
+def validate_mesos_container_log_sink(mesos_container_log_sink):
+    assert mesos_container_log_sink in ['journald', 'logrotate', 'journald+logrotate'], \
```

Comment:
Hm, this file seems to use backslashes for these cases.  For example, the code in [`validate_bootstrap_tmp_dir`](https://github.com/dcos/dcos/blob/86f630bda0763318a969e00f11d8ac85eca7cdfe/gen/calc.py#L410-L411)